backend/src/action/docker.rs

```diff
@@ -44,6 +55,32 @@ pub struct DockerAction {
     pub sigterm_timeout: Option<SerdeDuration>,
 }
 impl DockerAction {
+    pub fn validate(
```

Comment:
Opinion: We could have had this return a Validated<DockerAction>. That would make things that need the validation to be run to know that this function was ran.